template/.github/workflows/build.yml.j2

```diff
@@ -38,21 +38,14 @@ jobs:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
         with:
           submodules: recursive
-      - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # renovate: tag=v1.0.7
-        with:
-          profile: minimal
-          toolchain: "{[ rust_version }]"
-          override: true
+      - uses: dtolnay/rust-toolchain@{{ rust_version }}
```

Comment:
Does this work? The curly braces were with brackets before

Reply:
Well, that is a good question..... I thought this was ansible/Jinja...

Good catch. Paging @soenkeliebau

Reply:
We [changed](https://github.com/stackabletech/operator-templating/blob/main/playbook/update_repo.yaml#L68) the variable delimiting pattern for historic reasons. We used to apply this to rpmspec files, which also use {{ }} to delimit stuff and confused Ansible.

So correct is `{[ rust_version }]`